src/App.js

```diff
@@ -193,7 +193,7 @@ class App extends Component {
 
         {currentPanel === "results" && (
           <BodyContainer>
-            <ContainerLeft>
+              <ContainerLeft>
```

Comment:
this indentation doesn't look intentional?

Reply:
Good catch, thanks!